packages/trader/src/sass/app/_common/mobile-widget.scss

```diff
@@ -164,3 +164,21 @@
         color: var(--text-prominent);
     }
 }
+
+#duration_amount_selector {
```

Comment:
@henry-deriv  let's use classname selector instead of id selector in order to follow the best practices and not have to be forced into using !important to override this style later :) More info on this is for example in this article: https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4

Reply:
makes sense